cmd/pin.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+)
+
+func createPinRun(pinned bool) func(cmd *cobra.Command, args []string) {
+	return func(cmd *cobra.Command, args []string) {
```

Comment:
Could you factor the implementation into a normal function? I don't really like having the Run func returned like this, imo it'd be cleaner to have them inline in each command

Reply:
I wasn't sure if you wanted me to completely inline or just change createPinRun from a func generator to a normal function, so I did the latter.